src/main/java/com/comphenix/protocol/events/PacketListener.java

```diff
@@ -23,49 +23,53 @@
  * Represents a listener that receives notifications when packets are sending or being received.
  * <p>
  * Use {@link PacketAdapter} for a simple wrapper around this interface.
+ *
  * @author Kristian
  */
 public interface PacketListener {
-	
+
 	/**
 	 * Invoked right before a packet is transmitted from the server to the client.
 	 * <p>
 	 * Note that the packet may be replaced, if needed.
 	 * <p>
-	 * This method is executed on the main thread in 1.6.4 and earlier, and thus the Bukkit API is safe to use.
-	 * <p>
-	 * In Minecraft 1.7.2 and later, this method MAY be executed asynchronously, but only if {@link ListenerOptions#ASYNC}
-	 * have been specified in the listener. This is off by default.
+	 * This method is executed on the main server thread by default. However, some spigot forks (like paper) schedule
+	 * specific packets off the main thread. If the {@link ListenerOptions#ASYNC} option is specified any invocation of
```

Comment:
Should that be SYNC or ASYNC?

Reply:
Good catch!

Reply:
ASYNC, but the wording was off anyway :)